htdocs/commande/document.php

```diff
@@ -80,6 +80,10 @@
 if ($user->socid) {
 	$socid = $user->socid;
 }
+
+// Initialize technical object to manage hooks of page. Note that conf->hooks_modules contains array of hook context
+$hookmanager->initHooks(array('ordercarddocument'));
```

Comment:
according to modulebuilder, we should have here
`$hookmanager->initHooks(array($object->element.'document', 'globalcard')); // Note that conf->hooks_modules contains array
`